options/cf_options.h

```diff
@@ -242,4 +240,8 @@ struct MutableCFOptions {
 
 uint64_t MultiplyCheckOverflow(uint64_t op1, double op2);
 
+// Get the max file size in a given level.
+uint64_t MaxFileSizeForLevel(const MutableCFOptions& cf_options,
```

Comment:
sorry I might've missed something -- why'd we move this out of `MutableCFOptions` object?

Reply:
oh I just read the earlier review... oops sorry :p.